crates/bevy_render/src/mesh/mesh.rs

```diff
@@ -204,12 +213,39 @@ pub struct Mesh {
     indices: Option<Indices>,
 }
 
+/// Contains about geometry in the form of a triangle mesh.
```

Comment:
Nit: "contains geometry in the form of a mesh" (removed "about". and not all meshes are "triangles", they might also be lines)